.travis.yml

```diff
@@ -101,6 +102,20 @@ jobs:
           - CMAKE_ARGS="-DCMAKE_CXX_COMPILER=/usr/bin/c++ -DCMAKE_C_COMPILER=/usr/bin/cc"
           - CACHE_NAME=JOB3
 
+      - os: osx
+        osx_image: xcode11.6
+        language: cpp
+        cache: 
+         - ccache: true
+         - directories:
+            - $HOME/.ccache
+            - $HOME/Library/Caches/Homebrew
+            - /usr/local/Homebrew
+        env:
+          - CMAKE_OPTS="-DCMAKE_CXX_COMPILER_LAUNCHER=/usr/local/bin/ccache -DBUILD_QT5=ON -DUSE_PYTHON3=1 -DCMAKE_CXX_FLAGS='-std=c++11' -DBUILD_FEM_NETGEN=1 -DBUILD_FEM=1 -DCMAKE_PREFIX_PATH='/usr/local/Cellar/qt/5.15.1/lib/cmake;/usr/local/Cellar/nglib/v6.2.2007/Contents/Resources' -DBUILD_FEM_NETGEN:BOOL=ON -DFREECAD_USE_EXTERNAL_KDL=ON -DCMAKE_BUILD_TYPE=Release"
```

Comment:
Homebrew maintainer here, instead of `/usr/local/Cellar/qt/version` you probably want to use `/usr/local/opt/qt` otherwise you need to fix your Travis file with every upstream update you do.

Reply:
Hi, yes there is a couple of hard path which needs to be eradicated. I left them in, as I am still into a WIP mode regarding the various homebrew formulas that I updated a few days ago. My priority was to re-enable MacOS build into travis and issue a couple of upcoming  PR to polish these various issues, as MacOS build is reporting a few Warning which I believe we shall take care of. I can run the polishing before we integrate it, but I must admit that this is a pain to debug as the CCACHE is built each time I am upgrading the PR as the Master one is too old, ending up into crazy compilation time. I would like to overcome that issue by integrating this first, get a descent CCACHE which led to a couple of minutes instead of hours.

Reply:
Just updated these 2 paths, thanks to have reported them

Reply:
I just made 2 additionnal commit to clean up quickly these issues.